client/src/plugins/ajax.ts

```diff
@@ -55,7 +60,7 @@ export class AjaxRequest {
       ({ data, status }) => {
         if (status === 200) {
           this.store.commit("SET_USER", data.user);
-          if (data.user.isVerified) this.router.push({ name: LOBBY_PAGE });
+          if (data.user.isVerified) this.router.push({ name: FREE_PLAY_LOBBY_PAGE });
```

Comment:
Should we add another check to see if we should funnel them to the tournament dashboard instead?

Reply:
Oh yeah I suppose that would be one way to fix it, though may be confusing to route to one or the other when both are active. I think routing to home is fine until there's a way to pass a next param to the login routes